class-gf-razorpay.php

```diff
@@ -0,0 +1,336 @@
+<?php
+
+require_once ('razorpay-sdk/Razorpay.php');
+use Razorpay\Api\Api;
+
+GFForms::include_payment_addon_framework();
+
+class GFRazorpay extends GFPaymentAddOn
+{
+    protected $_version                  = GF_RAZORPAY_VERSION;
+    protected $_min_gravityforms_version = '1.9.3';
+    protected $_slug                     = 'razorpay-gravity-forms';
+    protected $_path                     = 'razorpay-gravity-forms/razorpay.php';
+    protected $_full_path                = __FILE__;
+    protected $_url                      = 'http://www.gravityforms.com';
+    protected $_title                    = 'Gravity Forms Razorpay Add-On';
+    protected $_short_title              = 'Razorpay';
+    protected $_supports_callbacks       = true;
+
+    // Permissions
+    protected $_capabilities_settings_page = 'gravityforms_razorpay';
+    protected $_capabilities_form_settings = 'gravityforms_razorpay';
+    protected $_capabilities_uninstall     = 'gravityforms_razorpay_uninstall';
+
+    // Automatic upgrade enabled
+    protected $_enable_rg_autoupgrade = true;
+
+    private static $_instance = null;
+
+    const GF_RAZORPAY_KEY    = 'gf_razorpay_key';
+    const GF_RAZORPAY_SECRET = 'gf_razorpay_secret';
+    const RAZORPAY_ENTRY_ID  = 'gf_razorpay_entry_id';
+    const RAZORPAY_ORDER_ID  = 'razorpay_order_id';
+
+    //cookie set for one day
+    const COOKIE_DURATION    = 86400;
+
+    public static function get_instance()
+    {
+        if (self::$_instance === null)
+        {
+            self::$_instance = new GFRazorpay();
+        }
+
+        return self::$_instance;
+    }
+
+    public function init_frontend()
+    {
+        parent::init_frontend();
+
+        add_action('gform_after_submission', array($this, 'pay_using_razorpay'), 10, 2);
+    }
+
+    public function plugin_settings_fields()
+    {
+        return array(
+            array(
+                'title'       => 'razorpay_settings',
+                'fields'      => array(
+                    array(
+                        'name'    => self::GF_RAZORPAY_KEY,
+                        'label'   => esc_html__('Razorpay Key', 'razorpay-gravity-forms'),
+                        'type'    => 'text',
+                        'class'   => 'medium',
```

Comment:
Why the additional spaces? Use an alignment plugin to align all the tabs in each block